cloudflare/package.json

```diff
@@ -11,28 +11,18 @@
     ]
   },
   "devDependencies": {
-    "@eslint/js": "^9.33.0",
     "@types/node": "^24.2.1",
-    "eslint": "^9.33.0",
     "prettier": "^3.6.2",
     "typescript": "5.8.3",
-    "typescript-eslint": "^8.39.1",
     "wrangler": "^4.28.1"
   },
-  "files": [
-    "./**/*.d.ts",
-    "./**/*.js"
-  ],
-  "license": "MIT",
-  "name": "htsget-deploy-cloudflare",
-  "packageManager": "pnpm@10.11.0",
+  "name": "htsget-workers",
   "scripts": {
-    "build": "tsc",
+    "build": "wrangler types && tsc",
```

Comment:
Types need to be generated before publishing so that there aren't errors in the package.